index.html

```diff
@@ -152,49 +152,50 @@ <h2>My Favourite Things</h2>
                             The logo for the math rock band "Polyphia"
                         </figcaption>
                     </figure>
-                    <input type="radio" name="favourite" id="tab_2" />
-                    <label for="tab_2">
-                        <h3>Bands</h3>
-                    </label>
-                    <div>
-                        <ol>
-                            <li>EDEN PROJECT</li>
-                            <li>Polyphia</li>
```

Comment:
This whole section was mostly ok, but you had your divs wrapped around the wrong elements in some cases,